scenario/BulkerScenario.ts

```diff
@@ -10,15 +10,16 @@ scenario(
     filter: async (ctx) => await isBulkerSupported(ctx) && !matchesDeployment(ctx, [{deployment: 'weth'}, {network: 'mumbai'}]),
     supplyCaps: {
       $asset0: 3000,
+      $asset2: 1,
```

Comment:
commented on this in #668, but wanted to mention it here; 

This scenario assumes that the wrapped native token collateral asset has a supply cap of at least .01 units.

In the Polygon deploy, that asset is $asset2